### ISSUE TITLE
Is it working on IOS? 

### ISSUE THREAD

**Issue by voody2506** [3ddelano/epic-online-services-godot#29]
Hello. Is it support IOS build?

**Comment by 3ddelano** [comment]
Yes epic online services supports IOS but I have not configured it as I don't have a way to test it.

**Comment by voody2506** [comment]
Thank you

**Comment by voody2506** [comment]
Should we use Native platrofm SDKs ? Why we just can't use C SDK on all platforms (including mobile) ?

**Comment by 3ddelano** [comment]
We don't have access to the source code for EOS for IOS. Epic Online services only gives us the compiled native library to be used on the platform.

**Comment by 3ddelano** [comment]
I will try to get an example for using EOSG plugin with IOS, it will need modifying the Godot xcode project a bit.

**Comment by voody2506** [comment]
Thanks
